extensions/pkg/webhook/webhook.go

```diff
@@ -90,7 +109,13 @@ func New(mgr manager.Manager, args Args) (*Webhook, error) {
 	logger.Info("Creating webhook")
 
 	return &Webhook{
-		Path:    args.Path,
-		Webhook: &admission.Webhook{Handler: handler, RecoverPanic: true},
+		Name:           args.Name,
+		Action:         actionType,
+		Selector:       args.Selector,
+		ObjectSelector: args.ObjectSelector,
+		Path:           args.Path,
+		Target:         args.Target,
+		Webhook:        &admission.Webhook{Handler: handler, RecoverPanic: true},
+		Types:          objTypes,
```

Comment:
Any reason provider is not passed?
```suggestion
		Provider:       args.Provider,
		Name:           args.Name,
		Action:         actionType,
		Selector:       args.Selector,
		ObjectSelector: args.ObjectSelector,
		Path:           args.Path,
		Target:         args.Target,
		Webhook:        &admission.Webhook{Handler: handler, RecoverPanic: true},
		Types:          objTypes,
```